tests/acceptance/features/webUISettingsMenu/settingsMenu.feature

```diff
@@ -43,4 +43,62 @@ Feature: add users
     Then the administrator should be able to see the last login of these users in the User Management page:
       | username | last login  |
       | user1    | seconds ago |
-      | user2    | never       |
\ No newline at end of file
+      | user2    | never       |
+
+  Scenario: administrator should be able to see password column of user
+    When the administrator enables the setting "Show password field" in the User Management page using the webUI
+    Then the administrator should be able to see the password of these users in the User Management page:
+      | username |
+      | user1    |
+      | user2    |
+
+  Scenario: administrator should not be able to see password column of user
+    When the administrator disables the setting "Show password field" in the User Management page using the webUI
+    Then the administrator should not be able to see the password of these users in the User Management page:
+      | username |
+      | user1    |
+      | user2    |
+
+  Scenario: administrator should be able to see quota of user
+    When the administrator enables the setting "Show quota field" in the User Management page using the webUI
+    Then the administrator should be able to see the quota of these users in the User Management page:
+      | username |   quota   |
+      | user1    |   Default |
+      | user2    |   Default |
+    And the administrator changes the quota of user "user1" to "Unlimited" using the webUI
+    And the administrator changes the quota of user "user2" to "5 GB" using the webUI
+    And the administrator logs out of the webUI
+    And user "user1" logs in using the webUI
+    And the user logs out of the webUI
+    And user "user2" logs in using the webUI
+    And the user logs out of the webUI
+    And user admin logs in using the webUI
+    And the administrator browses to the users page
+    Then the administrator should be able to see the quota of these users in the User Management page:
+      | username |   quota     |
+      | user1    |   Unlimited |
+      | user2    |   5 GB      |
+
+
+  Scenario: administrator should not be able to see quota of user
+    When the administrator disables the setting "Show quota field" in the User Management page using the webUI
+    Then the administrator should not be able to see the quota of these users in the User Management page:
+      | username |
+      | user1    |
+      | user2    |
+    And the administrator enables the setting "Show quota field" in the User Management page using the webUI
```

Comment:
We could delete the steps from here - I am not sure that they demonstrated anything more than the steps above.